docs/release.md

```diff
@@ -68,7 +68,7 @@ checkpoints:
 
 ### Prerelease
 
-The prerelease occurs on the first Tuesday of a sprint. During this stage, the
+The prerelease occurs on the last Monday of a sprint. During this stage, the
```

Comment:
I think its the first Monday, since sprints start on a Thursday?

Reply:
Current releases straddle the sprint end/beginning. Prerelease on tuesday, sprint ends wednesday, stable release on Thursday. I just moved the dates in the release radar earlier by 1 day, so it's the last Monday/Wednesday of the sprint.

Reply:
Oops, I had the sprint dates wrong on my calendar 🤦‍♀️